wevote_settings/models.py

```diff
@@ -630,7 +634,8 @@ def create_remote_request_history_entry(
         }
         return results
 
-    def remote_request_history_entry_exists(self, kind_of_action, google_civic_election_id,
+    @staticmethod
+    def remote_request_history_entry_exists(kind_of_action, google_civic_election_id,
```

Comment:
Since you are moving some of these variables to the next line, please put each variable on its own line with 8 characters indented.

Reply:
Oh gosh, I missed that one! Thanks!